binr/r2r/load.c

```diff
@@ -72,9 +74,13 @@ static char *read_string_val(char **nextline, const char *val, ut64 *linenum) {
 			R_LOG_ERROR ("End token must be \"EOF\", got \"%s\" instead", endtoken);
 			return NULL;
 		}
-		RStrBuf *buf = r_strbuf_new ("");
+
+    RVecChar buf;
```

Comment:
nope. just use strbuf to build strings

Reply:
nope. rstrbuf here makes r2r unusable if compiled with sanitize=address,undefined